golangci.yaml

```diff
@@ -0,0 +1,47 @@
+linters:
```

Comment:
Please rename this file to `.golangci-lint.yaml`

Then you won't have to use --config parameter